pkg/transport/transport.go

```diff
@@ -89,8 +91,11 @@ func (transport *Transport) DialNode(ctx context.Context, node *pb.Node, opts ..
 func (transport *Transport) DialAddress(ctx context.Context, address string, opts ...grpc.DialOption) (conn *grpc.ClientConn, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	dialOpt := transport.tlsOpts.DialOption(storj.NodeID{})
-	options := append([]grpc.DialOption{dialOpt, grpc.WithBlock(), grpc.FailOnNonTempDialError(true)}, opts...)
+	options := append([]grpc.DialOption{
+		transport.tlsOpts.DialOption(ctx, storj.NodeID{}),
+		grpc.WithBlock(),
+		grpc.FailOnNonTempDialError(true),
```

Comment:
same

Reply:
We still need these for the same reasons as before. The change to transport credentials just ensures that we get a non-temporary error that will cause grpc to stop trying to reconnect.

Reply:
The non-temporary error change has now been pushed into the peer certificate verification function wrapper.